src/views/AddBlog.vue

```diff
@@ -1,9 +1,126 @@
 <template>
-    <h1 class="Display-4 text-center">Add here</h1>
-</template>>
+    <div style="margin:0 auto">
+    <nav class="main_nav">
+        <ul style="display: inline-block;">
+            <li><a href="#">Home</a></li>
+            <li><a href="#">Edit</a></li>
+            <li><a href="#">Format</a></li>
+            <li><a href="#">Help</a></li>
```

Comment:
Why is this required? we already have a Navbar

Please run your changes in an updated branch